Cython/Utility/ModuleSetupCode.c

```diff
@@ -391,6 +391,12 @@
 /* Whether to use METH_FASTCALL with a fake backported implementation of vectorcall */
 #define CYTHON_BACKPORT_VECTORCALL (CYTHON_METH_FASTCALL && PY_VERSION_HEX < 0x030800B1)
 
+#if !defined(CYTHON_LARGE_FUNCTION_POINTERS)
+// This can be defined to force an alternate code-path for testing purposes
+// There's no other reason to use it
+#define CYTHON_LARGE_FUNCTION_POINTERS 0
+#endif
```

Comment:
If this is really only meant for testing, I'd rather not give it a public `CYTHON_*` name but an internal one, like `__Pyx_TEST_large_func_pointers` or so.